core/src/main/java/hudson/model/Computer.java

```diff
@@ -448,7 +448,6 @@ public final void launch() {
     /**
      * CLI command to reconnect this node.
      */
-    @CLIMethod(name="connect-node")
     public void cliConnect(@Option(name="-f",usage="Cancel any currently pending connect operation and retry from scratch") boolean force) throws ExecutionException, InterruptedException {
```

Comment:
Args4j should be moved to the javadoc - as they are no longer arguments that can be used by args4?